### PR TITLE
Added Floppa Command to make the Bot better 

### PR DESCRIPTION
**Pull request by Hotkeyyy**

### CODE REVIEW

File: src/main/kotlin/net/axay/blaubot/commands/Floppa.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+object Floppa : SlashCommand(
+    "floppa",
+    "get Floppa"
+) {
+    override suspend fun handleCommand(interaction: Interaction) {
+        interaction.acknowledge(true).followUp {
+            embed {
+                title = "Floppa"
+                image = "https://media.tenor.com/images/40401b6421797ab623387168d8ecc88d/tenor.gif"
+                description = "This is floppa and he it's better than bingus"
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/blaubot/commands/Floppa.kt]: fix typo please

**Review by jakobkmar — CHANGES_REQUESTED** [review]

File: src/main/kotlin/net/axay/blaubot/commands/Floppa.kt (hunk 1 above)
Comment (Hotkeyyy) [reply src/main/kotlin/net/axay/blaubot/commands/Floppa.kt]: Was soll ich da denn fixxen

Comment (jakobkmar) [reply src/main/kotlin/net/axay/blaubot/commands/Floppa.kt]: nicht `he it's` sondern `he is`

**Comment by realjarek** [comment]
If we have Floppa we also need Bingus!

**Review by jakobkmar — APPROVED** [review]

**Pull request merged**